### ISSUE TITLE
Possibility to set the exact number of users to spawn (instead weight)

### ISSUE THREAD

**Issue by EzR1d3r** [locustio/locust#1939]
### Is your feature request related to a problem? Please describe.

I needed a special query that would be executed once every 30 seconds, regardless of the total number of users created. The only solution I came up with is to create one instance of a user with such a request. To do this, it was necessary to count his weight according to the formula

```
@events.test_start.add_listener
def on_test_start(environment: Environment, **kwargs) -> None:
    SingleUserUser.weight = 1
    CommonUser.weight = environment.runner.target_user_count - 1
```
This is not very convenient not very obvious.

### Describe the solution you'd like

Just a field (like weight) "count" or "fix_count", which describes the exactly count of user instances. I guess weight parameter in that case have to be ignored.

### Additional context

If you find it a useful figure I can try to do

**Comment by cyberw** [comment]
Looks like a reasonable approach. If someone makes a PR with a few appropriate tests I’ll be happy to review.

**Comment by EzR1d3r** [comment]
I was thinking about the realization of this feature.

The approach of @marcinh is pretty laconic and understandable. But here is the some limit - the fixed count users spawns first. In some cases that behavior may not be wanted.

May be we can recalculate all the weights together? For example the approach based on desired instance count of every class. For example:

User1.fix_count = 5
User2.weight = 1
User3.weight = 2
tagret_count = 11

sum_weights_non_fix = User1.weight + User2.weight # 3

user1_weight = User1.fix_count # 5
user2_weight = (tagret_count - user1_weight) * (User2.weight/sum_weights_non_fix) # 2
user3_weight = (tagret_count - user1_weight) * (User3.weight/sum_weights_non_fix) # 4

Further act as usually: normalizing, smooth, etc...

It is possible to slightly increase the weight of fixed users to generate a little faster, so we can be sure the total number would be enough for them. Controlling the count somewhere higher in the hierarchy, has not yet been precisely determined, **_add_users_on_workers** may be.

**Comment by EzR1d3r** [comment]
> Looks like a reasonable approach. If someone makes a PR with a few appropriate tests I’ll be happy to review.

After diving into code I realized what my previous comment was pretty senseless. I have deleted it. )) After experiments, I also realized that in the current implementation of user spawning, trying to generate them more evenly, they still have to give large weights, although I achieved some success and even wrote part of the tests, my implementation close to generating fixed users at first. So I back to the @marcinh  realization, with some modification to resolve case with restore fixed users count after ramp-down and ramp-up.
